django/core/handlers/base.py

```diff
@@ -305,3 +297,64 @@ def apply_response_fixes(self, request, response):
         for func in self.response_fixes:
             response = func(request, response)
         return response
+
+    # LEGACY methods, remove after old style middlewares are removed.
+    def process_exception_by_middleware(self, exception, request):
```

Comment:
Should we rename this with `_old` in the name, for clarity where it's used that it is part of the old system?